packages/drivers/odsp-driver/src/test/prefetchSnapshotTests.spec.ts

```diff
@@ -202,17 +202,18 @@ describe("Tests for prefetching snapshot", () => {
 			const version = await service.getVersions(null, 1);
 
 			assert.deepStrictEqual(version, expectedVersion, "incorrect version");
-			assert(
-				mockLogger.events.filter((event) => event.eventName.includes("ObtainSnapshot_end"))
-					.length === 1,
-				"1 Obtain snapshot event should be there",
-			);
-			assert(
-				mockLogger.matchEvents([
-					{ eventName: "OdspDriver:ObtainSnapshot_end", method: "cache" },
-				]),
-				"Source should be cache",
-			);
+			// No longer able to check the logger that have been created locally
```

Comment:
I don't think we should check in commented-out test asserts. If they aren't valid, we should remove the code altogether. That said, I'm not sure we want to remove these cases? Presumably, if we restore the removed logger properties, we could restore these as well?

Reply:
Restored. Tong might have commented them during some intermediate state, right now they work fine.